src/MonitorProfiler/CMakeLists.txt

```diff
@@ -38,6 +38,9 @@ set(SOURCES
     Communication/CommandServer.cpp
     )
 
+# Include exceptions tracking feature
+#add_compile_options(-DDOTNETMONITOR_FEATURE_EXCEPTIONS)
```

Comment:
```suggestion
#add_compile_options(-DOTNETMONITOR_FEATURE_EXCEPTIONS)
```

Reply:
It is correct as-is. The `-D` specifies that it is a preprocessor definition.